### PR TITLE
Added datagrid-badges

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

File: husky_components/datagrid/decorators/table-view.js [hunk 1]
```diff
@@ -742,6 +750,7 @@ define(function() {
                 });
             }
             if (!!this.icons) {
+                content = this.addBatchesToCellContent(content, column, record);
```
Comment (marcelmoosbrugger) [inline husky_components/datagrid/decorators/table-view.js]: @wachterjohannes i think addBatchesToCellContent shouldn't be in if (!!this.icons) but in if (!!this.batches)

**Comment by wachterjohannes** [comment]
@marcelmoosbrugger fixed everything!

**Comment by danrot** [comment]
What is a batch?

**Comment by wachterjohannes** [comment]
@danrot oups (: spelling ... i wanted "badge"

**Comment by danrot** [comment]
@wachterjohannes It's also wrong in the code, and I still don't really know what it is :wink:

**Comment by wachterjohannes** [comment]
yeah thats right! i will change that ... in short badges are signs on the left side of the cell-content (needed for "ghost" language in the datagrid)

image

**Comment by wachterjohannes** [comment]
@marcelmoosbrugger @danrot renamed (:

**Pull request merged**